config.py

```diff
@@ -1,11 +1,21 @@
-import datetime
+import data_util
+from datetime import date, timedelta
+from sklearn import linear_model
+from statsmodels.regression.rolling import RollingOLS
```

Comment:
Try not to import these unused packages in config file, but instead import in the files that you need to use them.

The rationale is that you may have multiple files using config but not all using the imported packages here so you can optimize the import time spent.